### DISCUSSION TITLE
Is this still in development?

### DISCUSSION THREAD

**Discussion by santiagofn** [trakt-tools/universal-trakt-scrobbler#191] [Q&A] [answered]
Last activity I saw is from 3 months ago. Are there people still working in v1?

**Comment by MrMamen** [comment, marked as answer]
Well. It's an open source project so you're welcome to join. Its been mostly developed by @rafaelgomesxyz and to lesser degree myself. As you can see on the activity statistics below it has always had long periods where no one has done anything:
![image](https://user-images.githubusercontent.com/5688606/192946763-b272cd84-d529-45b5-b32f-32e6030f4e61.png)
Source: https://github.com/trakt-tools/universal-trakt-scrobbler/graphs/contributors

I made some work last week on a branch, so I haven't completely abandoned it. As a minimum I want to keep trying to support the streaming services that I use, and hopefully other developers will support their favorite services. 

However there is a huge looming threat noted in #156 that requires a huge rewrite. As far as I know, none is working on that ATM.

**Reply by rafaelgomesxyz** [reply]
I was working on the manifest v3 upgrade a while back. I'll see if I can resume that soon.

That's the biggest issue that's preventing a release on the Chrome/Firefox stores.

**Comment by rafaelgomesxyz** [comment]
I'm trying to get back to development on it soon. My job's just been getting in the way lately.

**Comment by W13N3N** [comment]
@rafaelgomesxyz , I hope you are doing well! First of all, thanks for this awesome tool!! I'm happy i came across this plugin since traktflix stopped working for me.

I'm curious if there is an estimate of when this plugin will be released to Chrome store?
